test/iterator-test.js

```diff
@@ -66,23 +66,17 @@ make('iterator optimized for seek', function (db, t, done) {
 
 make('close db with open iterator', function (db, t, done) {
   var ite = db.iterator()
-  var cnt = 0
-  var hadError = false
+  var nexted = false
 
   ite.next(function loop (err, key, value) {
-    if (cnt++ === 0) {
-      t.ifError(err, 'no error from next()')
-    } else {
-      t.equal(err.message, 'iterator has ended')
-      hadError = true
-    }
-    if (key !== undefined) { ite.next(loop) }
+    nexted = true
+    t.ifError(err, 'no error from next()')
+    // if (key !== undefined) { ite.next(loop) }
```

Comment:
Gotta restore the previous behavior here (calling `it.next()` after `db.close()` should throw).